### PR TITLE
Mainnet: 0.8 aragonOS, templates

### PR DESCRIPTION
**Pull request by izqui**

### CODE REVIEW

**Comment by izqui** [comment]
@sohkai we should merge this soon if the artifacts need to be fetched from IPFS for testing.

File: environments/mainnet/readme.md [hunk 1]
```diff
@@ -14,6 +14,70 @@
...
+### DAO Templates
+
+Deployed: `bare-template`, `company-template`, `company-board-template`, `membership-template`, `reputation-template`
+
+Command:
+```
+npm run publish:mainnet
+```
+
+`membership-template.aragonpm.eth` was deployed twice due to issues with transactions not propagating properly
```
Comment (sohkai) [inline environments/mainnet/readme.md]: 😢

File: environments/staging/readme.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ Final versions:
 - `bare-template.aragonpm.eth`: `1.0.0`
 - `company-template.aragonpm.eth`: `1.0.0`
 - `company-board-template.aragonpm.eth`: `2.0.0`
-- `membership-template.aragonpm.eth`: `1.0.0``
```
Comment (facuspagnuolo) [inline environments/staging/readme.md]: I think this was already fixed

File: environments/mainnet/readme.md (hunk 1 above)
Comment (facuspagnuolo) [reply environments/mainnet/readme.md]: Should we add the first version as well?

Comment (facuspagnuolo) [reply environments/mainnet/readme.md]: In `deployments.yml` I mean

**Pull request merged**